platform/admin/core/admin-core-api/src/main/java/org/codice/ddf/admin/core/api/ConfigurationAdmin.java

```diff
@@ -80,13 +79,6 @@
    */
   List<Service> listServices(String serviceFactoryFilter, String serviceFilter);
 
-  /**
-   * Returns all configurations that satisfy the provided filter.
-   *
-   * @see org.osgi.service.cm.ConfigurationAdmin#listConfigurations(String)
-   */
-  Configuration[] listConfigurations(String filter) throws IOException, InvalidSyntaxException;
```

Comment:
Why was this removed?

Reply:
It was only added on the last DDF-3185 PR because the plugin needed it. When we changed the design, it was no longer needed.